### PR TITLE
Version bumped for release

### PR DESCRIPTION
**Pull request by JEuler**

### CODE REVIEW

**Comment by techouse** [comment]
You might want to review the [analyzer constraint in chopper_generator/pubspec.yaml](https://github.com/lejard-h/chopper/blob/develop/chopper_generator/pubspec.yaml#L11). I had to constrain it to

```yaml
analyzer: ">=4.1.0 <4.3.0"
```

in order to satisfy the unit tests in the Github workflow. Maybe you could change it back to

```yaml
analyzer: ^4.1.0
```

and update the [Github workflow](https://github.com/lejard-h/chopper/blob/develop/.github/workflows/dart.yml#L73) so that it doesn't just update it to the latest version possible wich might cause the workflow test to fail.

File: chopper_built_value/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 1.1.0
+
+- Chopper upgraded
```
Comment (stewemetal) [inline chopper_built_value/CHANGELOG.md:5]: Does this update change the behavior or the API of the library? If not, then I think it's just a `1.0.1`

File: chopper/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 4.1.0
+
+- API breaking changes (FutureOr)
```
[the remark this answers is not in the document]
Comment (techouse) [reply chopper/CHANGELOG.md:5]: I agree. It should be `5.0.0`.

**Comment by JEuler** [comment]
Whoops, will update this tomorrow ahaha, and will check analyzer, thx. And thx for reviewing!

**Review by techouse — APPROVED** [review]

**Comment by JEuler** [comment]
Hmmmm, @stewemetal I think need your approval.

**Review by stewemetal — APPROVED** [review]
LGTM, go go go! 👍

**Pull request merged**